gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/JdbcOrganizationUserRepository.java

```diff
@@ -587,13 +599,29 @@ private Stream<JdbcOrganizationUser.Attribute> convertAttributes(User item, List
         return Stream.empty();
     }
 
-    private Mono<Integer> deleteChildEntities(String userId) {
-        Mono<Integer> deleteRoles = template.delete(JdbcOrganizationUser.Role.class).matching(Query.query(where(FK_USER_ID).is(userId))).all();
-        Mono<Integer> deleteRolesDynamicRoles = template.delete(JdbcOrganizationUser.DynamicRole.class).matching(Query.query(where(FK_USER_ID).is(userId))).all();
-        Mono<Integer> deleteAddresses = template.delete(JdbcOrganizationUser.Address.class).matching(Query.query(where(FK_USER_ID).is(userId))).all();
-        Mono<Integer> deleteAttributes = template.delete(JdbcOrganizationUser.Attribute.class).matching(Query.query(where(FK_USER_ID).is(userId))).all();
-        Mono<Integer> deleteEntitlements = template.delete(JdbcOrganizationUser.Entitlements.class).matching(Query.query(where(FK_USER_ID).is(userId))).all();
-        return deleteRoles.then(deleteRolesDynamicRoles).then(deleteAddresses).then(deleteAttributes).then(deleteEntitlements);
+    private Mono<Integer> deleteChildEntities(String userId, UpdateActions actions) {
```

Comment:
Let's make sure that for 4.0.0 and master we have `Mono<Long>` (due to the migration of the new bom dependencies)